### ISSUE TITLE
Regarding use of test set for model selection

### ISSUE THREAD

**Issue by vardaan123** [chou141253/FGVC-PIM#36]
In the code, you report the best accuracy on the test set over all epochs. But generally, the validation set should be used for model selection and not the test set. I am aware that the validation set is not available for CUB-200-2011. Is this the standard practice for this dataset? There are other works that use a subset of the training set for validation. In that case, the comparison may not be fair.
